File: Unidad3/Java/Programa19.java
package com.mycompany.programa19;
import java.util.Scanner;

public class Programa19 {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Ingrese la cantidad de números a analizar: ");
        int n = input.nextInt();

        int ceroCount = 0;
        int mayoresCero = 0;
        int menoresCero = 0;

        for (int i = 1; i <= n; i++) {
            System.out.print("Ingrese el número #" + i + ": ");
            int numero = input.nextInt();

            if (numero == 0) {
                ceroCount++;
            } else if (numero > 0) {
                mayoresCero++;
            } else {
                menoresCero++;
            }
        }

        System.out.println("Números iguales a cero: " + ceroCount);
        System.out.println("Números mayores a cero: " + mayoresCero);
        System.out.println("Números menores a cero: " + menoresCero);
    }
}

File: Unidad3/C#/Programa19.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Ingrese la cantidad de números a analizar: ");
        int n = int.Parse(Console.ReadLine());

        int ceroCount = 0;
        int mayoresCeroCount = 0;
        int menoresCeroCount = 0;

        for (int i = 1; i <= n; i++)
        {
            Console.Write("Ingrese el número #" + i + ": ");
            int numero = int.Parse(Console.ReadLine);

            if (numero == 0)
            {
                ceroCount++;
            }
            else if (numero > 0)
            {
                mayoresCeroCount++;
            }
            else
            {
                menoresCeroCount++;
            }
        }

        Console.WriteLine("Números iguales a cero: " + ceroCount);
        Console.WriteLine("Números mayores a cero: " + mayoresCeroCount);
        Console.WriteLine("Números menores a cero: " + menoresCeroCount);
    }
}
